demo/src/docs/ExpandablePanel.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+import Paper from "elevate-ui/Paper";
+import Typography from "elevate-ui/Typography";
+import ExpandablePanel from "elevate-ui/ExpandablePanel";
+
+const ExpandablePanelExample = ({
+  values,
```

Comment:
In our demo code, if we aren't using using any of these variables in props, we should remove them so they don't accidentally get copied and pasted when the code snippet is ready to be used by our users in one of their projects.